nixos/modules/virtualisation/amazon-config.nix

```diff
@@ -1,5 +1,3 @@
-{ config, pkgs, modulesPath, ... }:
-
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-image.nix" ];
+  imports = [ ./amazon-image.nix ];
```

Comment:
This should be `<nixpkgs/nixos/modules/virtualisation/amazon-image.nix>`, since this file is copied to /etc/nixos/configuration.nix.